src/header-utilities/is_container.hpp

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
SFINAE is a bit more involved and widely used than should exist in a particular comment block. 

Here's an introduction to the idea. I'm happy to sit down with you and do a few exercise to get you working with the concept. The last person I tutored was up and running in about 90 minutes.
https://jguegant.github.io/blogs/tech/sfinae-introduction.html